packages/Tag/package.json

```diff
@@ -22,6 +22,7 @@
     "@paprika/raw-button": "^1.0.10-next.0",
     "@paprika/stylers": "^1.1.4-next.0",
     "@paprika/tokens": "^1.1.3-next.0",
+    "@paprika/icon": "^2.1.3",
```

Comment:
The most latest version can be found here https://github.com/acl-services/paprika/blob/master/packages/Icon/package.json#L3

We should use the most latest version when adding a paprika dependency to a paprika package. So that changeset can manage the dependency between them correctly